src/getCoinCombination.test.js

```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+  test(`returns correct combination for 1 cent`, () => {
```

Comment:
while technically it is OK, in this particular task - You were suposed to use another structure :  
 ```
it(`should return array with '0' when '0' is entered`, () => {
    expect(getCoinCombination(0))
      .toEqual([0, 0, 0, 0]);
  });
```